src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-management.component.ts

```diff
@@ -37,6 +37,8 @@ export class TutorialGroupsManagementComponent implements OnInit, OnDestroy {
         private activatedRoute: ActivatedRoute,
         private alertService: AlertService,
         private tutorialGroupsConfigurationService: TutorialGroupsConfigurationService,
+
```

Comment:
Is that empty line intentional?

Reply:
Removed line in b8f23d8ad65347e0854cc20d0bc007a435aae692! Good catch !